arroyo-console/src/routes/pipelines/JobDetail.tsx

```diff
@@ -399,14 +419,26 @@ function OperatorDetail({
           };
         });
       });
+
+      backpressureData = Object.entries(node_metrics.subtasks).map(kv => {
+        return kv[1].backpressure.map(m => {
+          return {
+            label: kv[0],
+            x: new Date(Number(m.time) / 1000),
+            y: m.value,
```

Comment:
Note that I didn't add the randomness that the other metrics have. @mwylde what's up with that?

Reply:
It adds a bit of jitter that helps separate out the substasks on the graph

Reply:
I've removed the graph.